course_discovery/apps/api/v1/views/course_runs.py

```diff
@@ -124,6 +125,12 @@ def list(self, request, *args, **kwargs):
               type: integer
               paramType: query
               multiple: false
+            - name: organization
+              description: Filter course runs by sub organizations or edx organization.
+              required: false
```

Comment:
I believe for our case the organization filter will be required. So that we can make sure that every service is using the courses data related.